packages/ember-engines/package.json

```diff
@@ -105,6 +105,9 @@
     "ember-cli-version-checker": "^5.1.2",
     "lodash": "^4.17.11"
   },
+  "peerDependencies": {
+    "ember-source": "^3.12 || 4"
```

Comment:
Why we're adding it here?

Reply:
I don't understand the question, could you elaborate? There's a description on why it's needed in the commit / PR body..

Reply:
I understood the description in the PR body @anehx but I'm trying to understand why you put the `ember-source` in `peerDependencies` instead `devDependencies`. Is it a requirement to have `@embroider/macros` working?

Reply:
It's already defined in `devDependencies` which is correct. I added it to `peerDependencies` because the `dependencySatisfies` macro of `@embroider/macros` requires it as peer dependency to be reliable otherwise it won't work properly (e.g. in a monorepo). This is needed because of [this change](https://github.com/embroider-build/embroider/pull/1070) but it's not properly documented yet.

Reply:
Ok, I got it, but why we're putting 3.12 and 4 here? Since ember-engines are not compatible with ember 4.x yet, I think it's not right

Reply:
@villander looks like you are right, technically we should add v4 only once support is fixed

Reply:
> technically we should add v4 only once support is fixed
Confirm.

@anehx - Can you update this to be `^3.12.0` for now?

Reply:
@rwjblue done, I also updated `@embroider/macros` to the latest version